src/python/impactx/dashboard/Analyze/plotsMain.py

```diff
@@ -14,10 +14,10 @@
 from trame.widgets import matplotlib, plotly, vuetify
 from wurlitzer import pipes
 
+from ..simulation import run_simulation
```

Comment:
It is still a bit funny that `run_simulation` is called from an analysis sub-folder.